tests/python/cli/example_function.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2023 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from typing import List
+
+import cvat_sdk.auto_annotation as cvataa
+import cvat_sdk.models as models
+import PIL.Image
+
+spec = cvataa.DetectionFunctionSpec(
+    labels=[
+        cvataa.label_spec("car", 0),
+    ],
+)
+
+
+def detect(
+    context: cvataa.DetectionFunctionContext, image: PIL.Image.Image
+) -> List[models.LabeledShapeRequest]:
+    return [
+        cvataa.rectangle(0, [1, 2, 3, 4]),
+    ]
```

Comment:
How about changing the interface to be the following:
- The function does not return anything
- There is a function to add an annotation for a frame (e.g. `context.add_rectangle()`)

This way:
- the users won't need to know the return annotation types (there can be another function for the users who know)
- we can change the function API without breaking it
- we can manage annotation storage (and optimize it, potentially)

Reply:
It is a tempting proposal. 🤔 I'm just not sure if I'll have time to implement it; it's a pretty big change.